### ISSUE TITLE
plantuml couldn't preview 

### ISSUE THREAD

**Issue by imaojun** [tamlok/vnote#993]
OS Version (操作系统):  ArchLinux  Bspwm

VNote Version (Help - About VNote - Version: ) 2.7.2

Symptoms (表现):

I tried using plantuml in Vnote like the code snippet below, but when I changed to preview mode, plantuml couldn't preview it.

But when I am in editor mode, I can use right click to copy the graphic to png or svg. it works.

I tried using plantuml local jar or online server.The effects are the same as mentioned above.

BTW,how to use local preview in editor mode like mentioned in this article.
https://www.v2ex.com/t/451037#reply18

```puml
@startuml
Bob -> Alice : hello
@enduml
```
```puml
Bob -> Alice : hello
```

How to Repro (如何复现):

copy upper plantuml code to Vnote, and try to preview.

**Comment by tamlok** [comment]
Could you please paste your plantuml settings here?

**Comment by imaojun** [comment]
@tamlok I uploaded a gif. What can I do to solve this problem?
![Peek 2019-10-08 13-52](https://user-images.githubusercontent.com/28778042/66370472-069f8680-e9d3-11e9-904c-2a59366c3f52.gif)

**Comment by tamlok** [comment]
1. Do you install VNote from vnote-bin or AppImage? If not, please try that.
2. Do you restart VNote?

**Comment by imaojun** [comment]
![image](https://user-images.githubusercontent.com/28778042/66444045-3488d780-ea74-11e9-8859-2b3959c42461.png)

I tried uninstalling vnote-git from the archlinuxcn repository and reinstalling vnote-bin from the AUR repository, but the download from the Aur repository failed (requires a proxy). So I installed vnote-git again from archlinuxcn. 

It is working now.

However, when I changed from preview mode to vim editor mode, it got stuck. I can't move using the keyboard. I can only get back to normal by right clicking on it.

This is another question, so I opened a new issue.
